packages/core/src/components/VirtualizedList/__stories__/VirtualizedList.stories.helpers.tsx

```diff
@@ -2,17 +2,19 @@
 import React from "react";
 import { Tip } from "vibe-storybook-components";
 
-export const generateItems = (defaultItemSize = 30, itemsCount, layout) => {
-  const items = [];
+export const generateItems = (defaultItemSize: number = 30, itemsCount: number, layout: string) => {
```

Comment:
Please remove the type from `defaultItemSize` as it inferred from the default value
See [CI check](https://github.com/mondaycom/vibe/actions/runs/11410064924/job/31787409356?pr=2531#step:6:152)

Reply:
Made the change in the latest commit
please check if it is correct